client/analytics/report/products/table-variations.js

```diff
@@ -123,11 +110,14 @@ export default class VariationsReportTable extends Component {
 					value: orders_count,
 				},
 				{
-					display: (
-						<Link href={ editPostLink } type="wp-admin">
-							{ stockStatuses[ stock_status ] }
-						</Link>
-					),
+					display:
+						'instock' === stock_status && stock_quantity && stock_quantity <= low_stock_amount ? (
```

Comment:
Should we extract this logic into a util method since it is used twice in this PR alone? Also I'm thinking at some point we will need to suppress the low stock logic for sites that don't do inventory tracking. So if we had one util method that we based logic like this off of, it would be a singular spot to add tweaks like that.

Reply:
`isLowStock` has now been split into a util in https://github.com/woocommerce/wc-admin/pull/987/commits/0d4bd2e3f8b931c17a1eaed3bb0959f84910682c.

I placed it in a `/analytics/report/products/utils.js` file. Any opinion on that?